code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -408,7 +408,7 @@ Difficulty: Hard
 /obj/effect/decal/cleanable/blood/gibs/bubblegum/can_bloodcrawl_in()
 	return TRUE
 
-/mob/living/simple_animal/hostile/megafauna/bubblegum/do_attack_animation(atom/A, visual_effect_icon)
+/mob/living/simple_animal/hostile/megafauna/bubblegum/do_attack_animation(atom/A, visual_effect_icon, obj/item/used_item, no_effect)
```

Comment:
Does this fix anything, or are you just including the proper parameters because that's what the base proc uses?

Reply:
*Technically* this counts as a fix still since it should have had the arguments anyway, but the main reason is because dreamchecker got annoyed at me if I didn't add them:
![image](https://user-images.githubusercontent.com/57483089/126851973-6693193d-3aff-4cda-be7f-f99ab35c2dce.png)